### PR TITLE
Switch the base image to SLE Micro for Rancher

### PR DESCRIPTION
**Pull request by tjjh89017**
- Don't zypper install here
- Luet package: Replace utils/installer with toolchain/elemental-cli.
- Use rancherd included in baseOS

base to https://build.opensuse.org/package/show/isv:Rancher:Harvester:baseos/baseos

### CODE REVIEW

**Comment by tjjh89017** [comment]
link to https://github.com/harvester/harvester/issues/1933

File: .drone.yml [hunk 1]
```diff
@@ -42,6 +42,35 @@ steps:
     event:
       - push
 
+- name: docker-build-sle-micro
```
Comment (bk201) [inline .drone.yml:47]: Could we trigger PR build according to branches?
`dev` -> `docker-build`
`sle-micro` -> `docker-build-sle-micro`

Comment (tjjh89017) [reply .drone.yml:47]: I think I forgot some "when" conditions.
let me deal with it.

Comment (tjjh89017) [reply .drone.yml:47]: let me check it.

Comment (tjjh89017) [reply .drone.yml:47]: It should work now.
This PR is built by `docker-build-sle-micro` task

**Review by bk201 — APPROVED** [review]
lgtm!

**Pull request merged**